cms/core/utils.py

```diff
@@ -75,3 +81,17 @@ def latex_formula_to_svg(latex: str, *, fontsize: int = 18, transparent: bool =
         svg_string = "\n".join(svg_string.split("\n")[3:])
 
     return svg_string
+
+
+def redirect_to_parent_listing(
+    page: "Page", request: "HttpRequest", get_listing_url_method: str, redirect_status: int = 307
```

Comment:
For clarity and not to make it sound confuing with an actual method, this might be better named as: `listing_url_attr`, `listing_url_method_name`

Reply:
Done! [e5074e2](https://github.com/ONSdigital/dis-wagtail/pull/387/commits/e5074e20359dff4e9e2b49dd8d3146a8bbfb5748)